base/docs/helpdb/Base.jl

```diff
@@ -6608,10 +6608,11 @@ Convolution of two vectors. Uses FFT algorithm.
 conv
 
 """
-    unsafe_store!(p::Ptr{T},x,i::Integer)
+    unsafe_store!(p::Ptr{T}, x, [i::Integer], [align::Integer])
```

Comment:
Why not use `=1` for arguments instead of brackets (as I realized a bit late)? That would be clearer, especially as regards the default value of `i`.